typespec/UCP/aws-credentials.tsp

```diff
@@ -63,6 +63,9 @@ model AwsPlaneNameParameter {
 enum AWSCredentialKind {
   @doc("The AWS Access Key credential")
   AccessKey,
+
+  @doc("The AWS IRSA credential")
```

Comment:
I'd suggest rephrasing it and making descriptions a bit more verbose for clarity. IRSA itself is not a credential but a role with policies that provides the ability to manage credentials.

Here's a suggestion, but I'd get @Reshrahim's inputs as well since these descriptions are user facing.

```suggestion
  @doc("AWS IAM roles for service accounts. For more information, please see: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html")
```

Reply:
confirmed with @Reshrahim and added in this change.